straxen/plugins/event_shadow.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import pandas as pd
+
+import strax
+import straxen
+
+
+straxen.print_versions(('strax', 'straxen', 'cutax'))
+import warnings
+export, __all__ = strax.exporter()
+
+import warnings
+export, __all__ = strax.exporter()
+
+@export
+class Shadow(strax.Plugin):
+    """Compute several new parameters to describe the noise level.
+        We can use these to reduce AC BG. """
+
+    depends_on = ('event_basics','peak_basics','peak_positions')
+    parallel = False
+    provides = 'event_shadow'
+    __version__ = '0.0.5'
+
+    def infer_dtype(self):
+
+        dtype = [
+                ('pre_s2_area', np.float32,'previous s2 area [PE]'),
+                 ('shadow_dt', np.int64,'time diffrence to the previous s2 [ns]'),
+                 ('shadow', np.float32,'previous s2 shadow [PE/ns]'),
+                ('pre_s2_x', np.float32,'x of previous s2 peak causing shadow [cm]'),
+                 ('pre_s2_y', np.float32,'y of previous s2 peak causing shadow [cm]'),
+                 ('shadow_distance', np.float32,'distance to the previous s2 peak causing the max shadow [cm]')           
+                ]
+        dtype += strax.time_fields
+        return dtype
+
+    def compute(self, events, peaks):
+        #set a time window to look for peaks
+        roi_dt = np.dtype([(('back in time', 'time'), int),
+                                         (('till it begin','endtime'), int)])
+        roi = np.zeros(len(events), dtype=roi_dt)   
+        n_seconds = int(3e9)
+        roi['time'] = events['time'] - n_seconds
+        roi['endtime'] = events['time']
+        split_try = strax.split_touching_windows(peaks, roi, window=0)
```

Comment:
I think it is already by default `window=0` so you can remove the keyword argument.